source/chapters/user_interface.rst

```diff
@@ -56,15 +56,12 @@ Track Information Display
 
    The track information section of the deck
 
-.. versionadded:: 2.0
-   Parse the title and the artist from the file name when the corresponding
-   :term:`metadata` is missing.
-
 **Track Title**
   The title of the track that was loaded into a deck is displayed on top. This
   is the same as the title listed under the :guilabel:`Title` column heading in
   the Mixxx library. This information is initially loaded from the track's
-  :term:`metadata`.
+  :term:`metadata`. Is the metadata missing, parse the title and the artist
```

Comment:
"If the metadata is missing, the title and artist are guessed from the filename."